include/deal.II/lac/constraint_matrix.h

```diff
@@ -740,6 +740,14 @@ class ConstraintMatrix : public Subscriptor
                               VectorType                   &global_vector,
                               const FullMatrix<LocalType>  &local_matrix) const;
 
+  template <typename VectorType, typename LocalType>
+  void
+  distribute_local_to_global (const Vector<LocalType>      &local_vector,
```

Comment:
I think it would be nice to have a piece of documentation in front of this declaration. It can probably say something of the sort "Same as the previous function, except ...".